aries-site/src/pages/_app.js

```diff
@@ -59,15 +60,161 @@ const backgroundImages = {
+              notificationHeader.length;
+            //the position of the first bracket containing either new/updates is 22 characters away
+            //this includes the notification header and the \r\n\r\n that follow
+            const notificationList = prDescription
+              .slice(indexOfFirstComponent)
+              .split('\r\n\r\n');
+            //splits them into an array jumping between name, sections, and description
+            //like: ['[New]Header', ['Usage', 'Some Section'], '[The description for header]', '[Update]Button', ['Dos and Donts'], '[Description for button]']
+            const regExp = /\[([^)]+)\]/;
+            for (let j = 0; j < Object.keys(notificationList).length; j += 3) {
+              //+= 3 so it can jump between component descriptions
+              let typeChangeAndName = notificationList[j].trim();
+              let typeChange = regExp.exec(typeChangeAndName)[1].trim();
+              let pageName;
+              if (typeChange === 'Update') {
+                pageName = typeChangeAndName.slice('8').trim();
+              } else if (typeChange === 'New') {
+                pageName = typeChangeAndName.slice('5').trim();
+              }
+              if (pageName && !(pageName in nextHistory)) {
+                let sectionArray = notificationList[j + 1]
+                  .slice(1, -1)
+                  .split('][');
+                //to ensure there is proper capitlization for the section headers
+                for (let i = 0; i < Object.keys(sectionArray).length; i++) {
+                  sectionArray[i] =
+                    sectionArray[i].charAt(0).toUpperCase() +
+                    sectionArray[i].slice(1).toLowerCase();
+                }
+
+                let anchorLink = '';
+                if (Object.keys(sectionArray).length === 1) {
+                  //add an active link if only one section has been updated
+                  anchorLink =
+                    '#' +
+                    sectionArray[0].trim().replace(/\s+/g, '-').toLowerCase();
+                }
+
+                let newUpdate;
+                localStorageKey = `${pageName
+                  ?.toLowerCase()
+                  .replace(/\s+/g, '-')}-last-visited`;
+                if (window.localStorage.getItem(localStorageKey)) {
+                  newUpdate =
+                    window.localStorage.getItem(localStorageKey) >
+                    new Date(data[i].merged_at).getTime()
+                      ? false //if the last visit is more recent than the reported update, dont show it
+                      : true;
+                } else {
+                  //have never visited the page before
+                  newUpdate = true;
+                }
+                nextHistory[pageName] = {
+                  changeKind: typeChange,
+                  description: notificationList[j + 2].slice(1, -1),
+                  date: data[i].merged_at,
+                  sections: sectionArray,
```

Comment:
Should `sectionArray` just be called `sections`? Generally we don't include the variable type (array, bool, etc.) in the variable name